src/sttn/eval/analyst_eval.ipynb

```diff
@@ -0,0 +1,576 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "############################################################################\n",
+    "## VARIABLES \"dataset_name\", \"exp_prefix\", \"exp_version\" AND \"model_name\" ##\n",
+    "## WILL BE CREATED BY THE nbconvert preprocessor ABOVE                    ##\n",
+    "############################################################################"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Upload libraries and initialize env variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sttn.nli.analyst import STTNAnalyst\n",
+    "from langchain.globals import set_debug\n",
+    "from langsmith import Client\n",
+    "from langsmith.schemas import Example, Run\n",
+    "from langsmith.evaluation import evaluate, LangChainStringEvaluator\n",
+    "import backoff\n",
+    "import openai\n",
+    "\n",
+    "from typing import List\n",
+    "\n",
+    "set_debug(False)\n",
+    "\n",
+    "# Langsmith client\n",
+    "client = Client()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Upload `langsmith dataset`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_dataset = client.read_dataset(dataset_name=dataset_name)\n",
```

Comment:
Where are the datasets stored? Can we include them in this PR?

Reply:
They are stored in LangSmith